pype/hooks/nukestudio/pre_launch_args.py

```diff
@@ -0,0 +1,27 @@
+import os
+from pype.lib import PreLaunchHook
+
+
+class NukeStudioLaunchArguments(PreLaunchHook):
+    order = 0
+    hosts = ["nukestudio"]
```

Comment:
`nukestudio` host name had been renamed to `hiero` and it apply to the name of folder too.